test/fixtures/specs/issueAndTms.ts

```diff
@@ -1,5 +1,8 @@
 import { suite, test } from 'mocha-typescript';
-import { issue, testCaseId } from '../../../index';
+import { issue, setPmsUrl, setTmsUrl, testCaseId } from '../../../index';
+
+setPmsUrl('http://pms-url');
```

Comment:
I believe you can move these 2 into `before` hook in a class body. It'll be more natural assuming OO-style.